ambari-server/src/main/python/ambari_server/serverUpgrade.py

```diff
@@ -123,8 +123,12 @@ def change_objects_owner(args):
 
 def run_schema_upgrade(args):
   db_title = get_db_type(get_ambari_properties()).title
-  confirm = get_YN_input("Ambari Server configured for %s. Confirm "
+  if not get_silent():
+    confirm = get_YN_input("Ambari Server configured for %s. Confirm "
                         "you have made a backup of the Ambari Server database [y/n] (n)? " % db_title, False)
+  else:
+    confirm = get_YN_input("Ambari Server configured for %s. Confirm "
+                        "you have made a backup of the Ambari Server database [y/n] (y)? " % db_title, True)
```

Comment:
@jaysensharma Thanks for the change.  Codewise I'd prefer reducing duplication a bit.  How about something like this:

```
  silent = get_silent()
  default_answer = 'y' if silent else 'n'
  default_value = silent
  confirm = get_YN_input("Ambari Server configured for %s. Confirm "
                         "you have made a backup of the Ambari Server database [y/n] (%s)? " % (db_title, default_answer), default_value)
```

Reply:
@adoroszlai  Thanks, I have made the suggested changes.